app/views/unattended/provisioning_templates/PXEGrub2/kickstart_default_pxegrub2.erb

```diff
@@ -20,26 +20,13 @@ test_on:
 - host6static
 -%>
 # This file was deployed via '<%= template_name %>' template
-<%
-  rhel_compatible = @host.operatingsystem.family == 'Redhat' && @host.operatingsystem.name != 'Fedora'
-  os_major = @host.operatingsystem.major.to_i
-
-  if (rhel_compatible && os_major < 7) || !@host.pxe_loader_efi?
```

Comment:
We can't remove this, we still support provisioning of [RHEL6 until 30 Jun 2024](https://endoflife.date/rhel)